src/helm/proxy/clients/auto_client.py

```diff
@@ -246,12 +247,18 @@ def get_toxicity_classifier_client(self) -> PerspectiveAPIClient:
 
     def get_critique_client(self) -> CritiqueClient:
         """Get the critique client."""
-        if not self.critique_client:
-            surgeai_credentials = self.credentials.get("surgeaiApiKey", None)
-            if surgeai_credentials:
-                self.critique_client = SurgeAICritiqueClient(surgeai_credentials, self._build_cache_config("surgeai"))
-            # To use the RandomCritiqueClient for debugging, comment out `raise ValueError` and uncomment the following
-            # from .critique_client import RandomCritiqueClient
-            # self.critique_client =  RandomCritiqueClient()
-            raise ValueError("surgeaiApiKey credentials are required for SurgeAICritiqueClient")
+        critique_type = self.credentials.get("critiqueType")
+        if critique_type == "random":
+            self.critique_client = RandomCritiqueClient()
+        elif critique_type == "mturk":
+            self.critique_client = MechanicalTurkCritiqueClient()
```

Comment:
We don't use the cache. When we import, we load the request results into memory instead of loading it into the cache. This is mostly because our current `Cache` abstraction does not support modifying existing entries, and the results will change over time (e.g. if requesting 2 respondents, the result may initially have just 1 respondent if only one has responded so far, and later on it may have 2 responses after a second respondent has responded). Note that this means that your memory has to be large enough to hold all the request and request result pairs.